fe/fe-core/src/main/java/com/starrocks/catalog/TableFunctionTable.java

```diff
@@ -236,7 +236,8 @@ private void parseFiles() throws DdlException {
             }
         } catch (UserException e) {
             LOG.error("parse files error", e);
-            throw new DdlException("failed to parse files", e);
+            String errorMsg = "failed to parse files. You may give a wrong url, please check it. " + path;
```

Comment:
1. Put the error message in ErrorCode.java file.
2. "Failed to parse the path: ${path}. The URL scheme (${scheme}) in the path is not supported."

I'm curious why do you put the testing code in the src dir?